WordPress-iOS-Shared/Core/WPStyleGuide.m

```diff
@@ -139,111 +140,120 @@ + (UIFont *)tableviewSectionHeaderFont
     return [WPFontManager openSansBoldFontOfSize:12.0];
 }
 
+
 #pragma mark - Colors
+// https://wordpress.com/design-handbook/colors/
 
-+ (UIColor *)wordPressBlue
++ (UIColor *)colorWithR:(NSInteger)red G:(NSInteger)green B:(NSInteger)blue alpha:(CGFloat)alpha
```

Comment:
Why did we create this method? It seems like it saves a few characters but I think it makes readability worse in the file.